charts/backingservices/README.md

```diff
@@ -53,7 +50,7 @@ srs:
   srsRuntime:
     #srs-service values
     replicaCount: 2
-    srsImage: platform-services/search-n-reporting-service:1.6.1
+    srsImage: platform-services/search-n-reporting-service:1.9.0-4
```

Comment:
@pega-kshev Can we put a place holder for image name here instead of platform-services like [this](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/values.yaml#L242)

Reply:
I think it makes sense, we are having instructions where we are suggesting them to rename the images after pulling in. So, should I name it `YOUR_SRS_IMAGE:TAG`? or `my-pega-srs:8.6.0` since that is what is being mentioned in the runbooks?

What do you think @taz-pega-work

Reply:
this makes sense to me too

Reply:
This is done now in https://github.com/pegasystems/pega-helm-charts/pull/265/commits/27be4781bbd5d990047b2815a13d12e2bae9e9e1